cloud/cloud_file_deletion_scheduler.h

```diff
@@ -51,7 +54,7 @@ class CloudFileDeletionScheduler
 
   mutable std::mutex files_to_delete_mutex_;
   std::unordered_map<std::string, int> files_to_delete_;
-  std::chrono::seconds file_deletion_delay_ = std::chrono::hours(1);
```

Comment:
Keeping the default value as 1 hour seems fine here? Especially since it is a primitive type so I don't think it gets default initialized to any value?

Reply:
good point!

Reply:
There is no constructor that doesn't init `file_deletion_delay_`, so keeping the init here would be just dead code.